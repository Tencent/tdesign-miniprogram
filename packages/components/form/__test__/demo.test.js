/**
 * 该文件为由脚本 `npm run test:demo` 自动生成，如需修改，执行脚本命令即可。请勿手写直接修改，否则会被覆盖
 */

import path from 'path';
import simulate from 'miniprogram-simulate';

const mapper = ['horizontal', 'vertical'];

describe('Form', () => {
  mapper.forEach((demoName) => {
    it(`Form ${demoName} demo works fine`, () => {
      const id = load(path.resolve(__dirname, `../../form/_example/${demoName}/index`), demoName);
      const container = simulate.render(id);
      container.attach(document.createElement('parent-wrapper'));
      expect(container.toJSON()).toMatchSnapshot();
    });
  });
});
