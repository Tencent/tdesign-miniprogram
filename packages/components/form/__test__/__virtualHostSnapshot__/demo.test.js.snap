// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form horizontal demo works fine 1`] = `""`;

exports[`Form vertical demo works fine 1`] = `""`;
